src/ServiceProvider.php

```diff
@@ -40,7 +40,7 @@ public function boot()
                 $encrypter = $app->make(Encrypter::class);
                 $table = $config->get('laravel-rollout.table');
 
-                $repository = new Repository(new DatabaseStore($connection, $encrypter, $table));
+                $repository = new Repository(new DatabaseStore($connection, $table));
```

Comment:
I don't think this will work with versions of Laravel < 5.5 since the encrypter is required for the database cache in those versions.

Reply:
That may be correct, but as Laravel 5.5 is a LTS version, I think it should be supported. Maybe in a different branch?

Reply:
Agreed. I'd like to see if we can continue to support 5.3 and 5.4 in addition to adding support for 5.5.